Server/Bff.WebApi.Administrations/DataAccess/Mysql/AdministrationContext.cs

```diff
@@ -25,6 +20,12 @@ protected override void OnConfiguring(DbContextOptionsBuilder options)
             options.UseMySql(connectionString, ServerVersion.AutoDetect(connectionString));
         }
 
-        public DbSet<User>? User { get; set; }
+        public DbSet<User>? Users { get; set; }
```

Comment:
Volgens mij mis ik hier de groep entity

Reply:
Opgelost in [deze commit](https://github.com/BFF-Unlimited/bff-unlimited/pull/24/commits/e393056bd5c54527859daa84facd492a12dc8906)